json-schema-converter/src/main/java/io/confluent/connect/json/JsonSchemaConverter.java

```diff
@@ -47,6 +47,7 @@
   private Deserializer deserializer;
 
   private boolean isKey;
+  private String keyOrValueSuffix;
```

Comment:
We don't need to track whether this was key or value, typically the topic name will already have that info.